Applications/Spire/Source/Ui/ListView.cpp

```diff
@@ -449,6 +449,8 @@ void ListView::on_list_operation(const ListModel::Operation& operation) {
 void ListView::on_current(const boost::optional<int>& current) {
   if(current) {
     m_items[*current]->m_item->setFocus();
+  } else {
+    setFocus();
```

Comment:
Where in the spec is this documented exactly? I can't see it in the user flow.

Reply:
It is in drafts\ui_kit\list_view, and it hasn't been moved to final.